presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -306,13 +306,24 @@ Property Name                                        Description
                                                      running in EC2, or when the catalog is in a different region.
                                                      Example: ``us-east-1``
 
+``hive.metastore.glue.endpoint-url``                 Glue API endpoint URL (optional). Useful when all requests
+                                                     to Glue should be proxied.
+                                                     Example: ``https://glue.us-east-1.amazonaws.com``
```

Comment:
I am not sure "be proxied" is the right term, as this implies they would end up hitting AWS APIs somehow, whereas this could be used to provide your own standalone glue implementation.

I think "Glue API endpoint URL" should be enough for now.

Reply:
Changed. That makes sense.